_pages/python-packages.md

```diff
@@ -25,6 +25,18 @@ redirect_from:
 To date, {{ total_packages }} packages have been accepted into the pyOpenSci
 ecosystem. [Click here to check out packages that are currently under review.](https://github.com/pyOpenSci/software-submission/issues)
 
+### Community Partners
+
+To view packages affiliated with our partner communities that are a part of our peer review partnership program, click on a community link below. [Learn more about our community partnerships here.](#)
+
+<a href="/partners/astropy.html"><button class="button community">Astropy</button></a>
```

Comment:
Should this link be `href="/communities/astropy.html"`

Reply:
oh i like using the word community in the url over partners. yes let's update it. we will want to update the yaml as well for the page. I will do that locally!!